packages/graph/hash_graph/lib/graph/src/ontology/mod.rs

```diff
@@ -351,6 +331,56 @@ impl PersistedEntityType {
     }
 }
 
+pub trait PersistedOntologyType {
```

Comment:
I remember when I wanted to make this ages ago and it got deleted 😁

Reply:
But we already have this, I just moved it 🙃